tests/ipython/test_ipython.py

```diff
@@ -297,3 +379,159 @@ class MockKedroContext:
         log_messages = [record.getMessage() for record in caplog.records]
         expected_message = f"Updating path to Kedro project: {updated_path}..."
         assert expected_message in log_messages
+
+
+class TestLoadNodeMagic:
+    def test_load_node_magic(self, mocker, dummy_function_file_lines, dummy_pipeline):
+        mock_jupyter_console = mocker.MagicMock()
+        mocker.patch("ipylab.JupyterFrontEnd", mock_jupyter_console)
+        mocker.patch(
+            "builtins.open", mocker.mock_open(read_data=dummy_function_file_lines)
+        )
+        pipelines.configure("dummy_pipeline")  # Setup the pipelines
+        my_pipelines = dummy_pipeline
+
+        def my_register_pipeline():
+            return my_pipelines
+
+        mocker.patch.object(
+            pipelines,
+            "_get_pipelines_registry_callable",
+            return_value=my_register_pipeline,
+        )
+
+        node_to_load = "dummy_node"
+        try:
+            magic_load_node(node_to_load)
+        except Exception:
+            # Fail if an error is thrown
+            assert False
```

Comment:
What does this try/catch do?

Reply:
@AhdraMeraliQB I think this can be removed? If a test run successfully `pytest` treat it as a success so `assertion` is not needed.